redash/handlers/query_results.py

```diff
@@ -64,7 +64,7 @@ def run_query_sync(data_source, parameter_values, query_text, max_age=0):
         return None
 
 
-def run_query(data_source, parameter_values, query_text, query_id, max_age=0):
+def run_query(data_source, parameter_values, query_text, query_id, max_age=0, parameter_schema=None):
```

Comment:
You removed the handling of the `None` case here, because I assume that the callers will always pass a `parameter_schema`? If that's the case, then should probably remove the default value.

Reply:
This is actually intentional. Handling of `None` is done in `ParameterizedQuery`.

Reply:
I thought that's the case, but didn't find it. Apparently I was looking in the wrong place. 🤦🏻‍♂️